packages/common/src/FormattingUtils.js

```diff
@@ -2,6 +2,7 @@ const networkUtils = require("./PublicNetworks");
 
 const BigNumber = require("bignumber.js");
 const moment = require("moment");
+const assert = require("assert");
```

Comment:
This was causing my local test to throw, not sure why it wasn't caught in CI?

Reply:
@daywiss FYI

Reply:
thats really wierd. might not be able to build on front end, its a node module

Reply:
oh im not sure why it wouldnt run locally though... its like a standard thing. unless it was actually asserting some thing to be true

Reply:
yeah you just have to import the assert module though